lib/docker-delta.coffee

```diff
@@ -10,7 +10,7 @@ btrfs = require './btrfs'
 utils = require './utils'
 Docker = require 'docker-toolbelt'
 
-docker = new Docker()
+docker = new Docker(socketPath: process.env.DOCKER_SOCKET ? '/var/run/docker.sock')
```

Comment:
Why is it called `DOCKER_HOST`? That makes me think it could be `http://...`, which wouldn't work if provided as a `socketPath`